filament/include/filament/RenderableManager.h

```diff
@@ -486,7 +486,7 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
             MorphTargetBuffer* morphTargetBuffer, size_t offset, size_t count);
 
     void setMorphTargetBufferAt(Instance instance, uint8_t level, size_t primitiveIndex,
-            MorphTargetBuffer* morphTargetBuffer, size_t count); //!< \overload
+            MorphTargetBuffer* morphTargetBuffer); //!< \overload
```

Comment:
make this one just an `inline` right there. No need to create a function for it.  We're just going to say that it's a helper. The "real" API is the one that takes the offset and count.

Reply:
This way is good to add utility functions. :)